src/App/examples/Example15/Example15.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {
+  ButtonBack,
+  ButtonFirst,
+  ButtonLast,
+  ButtonNext,
+  CarouselProvider,
+  Slide,
+  Slider,
+  ImageWithZoom,
+  DotGroup,
+} from '../../..';
+
+import s from '../../style.scss';
+
+export default () => (
+  <CarouselProvider
+    visibleSlides={1}
+    totalSlides={8}
+    step={1}
+    naturalSlideWidth={400}
+    naturalSlideHeight={500}
+    interval={3000}
+    intervalList={[10000, 9000, 8000, 7000, 6000, 5000, 4000, 3000]}
```

Comment:
@Bracciata I really like this idea for our Carousel. One thing I noticed is the timings are not intrinsically linked to a specific slide. For example, if I am working on this example and manually move from slide 2 where the interval is `9000` to slide 6 where the interval is `5000`. I would expect the slide 6 to have the interval of `5000` however the interval used would be for slide 3 of `8000`.  

Would you have an idea on what a fix might look like for this?

Reply:
That is a great question. It has been a while since I have heavily looked at the code and I didn't realize that bug existed. I am looking into it now!

Reply:
Possible solution mentioned at https://github.com/express-labs/pure-react-carousel/pull/365#discussion_r658855205
Working on implementing this now!